vms/platformvm/txs/executor/proposal_tx_executor.go

```diff
@@ -45,7 +47,8 @@ var (
 type ProposalTxExecutor struct {
 	// inputs, to be filled before visitor methods are called
 	*Backend
-	Tx *txs.Tx
+	BlkFeeManager *commonfees.Manager
```

Comment:
`BlkFeeManager` must be common to all transactions in a block, to duly calculate block complexity. This is why is passed to the tx executor by the caller